package.json

```diff
@@ -143,6 +147,8 @@
     "chance": "^1.0.18",
     "dotenv": "^7.0.0",
     "dotenv-webpack": "^1.7.0",
+    "eslint": "^5.16.0",
+    "eslint-plugin-react": "^7.12.4",
```

Comment:
You don't need to have this one, it's already a part of `@strv/eslint-config-react`

Reply:
@dannytce `@strv/eslint-config-react` was installing version `7.11.1` of `eslint-plugin-react`, which lacked some configuration for a rule a honestly don't remember which. Given the semantic versioning constraint, I expected both to be equivalent, but that did no happen to me, and I didn't dive deeper.

I'll give a try again on removing that and see how it goes ;)